packages/demand-bloc-quotes/src/transformer.ts

```diff
@@ -0,0 +1,59 @@
+import { get } from 'lodash'
+import { QuoteItem as OriginalQuoteItem } from '@karhoo/demand-api'
+
+export interface QuoteItem {
+  currencyCode: string
+  eta?: number | null
+
+  finalPrice?: null
```

Comment:
Should this empty line be here?

Reply:
I've taken itt from our existent quote transformer. Will prefer to keep this line just in case :)